README.md

```diff
@@ -61,4 +61,28 @@ Visit ['https://trello.com/signup'](https://trello.com/signup) in your browser a
 
 Step 2
 Create a new TRELLO API KEY and TOKEN
-Visit ['https://developer.atlassian.com/cloud/trello/guides/rest-api/api-introduction/'](https://developer.atlassian.com/cloud/trello/guides/rest-api/api-introduction/) for detailed instructions on how to both create an API_KEY and TOKEN
\ No newline at end of file
+Visit ['https://developer.atlassian.com/cloud/trello/guides/rest-api/api-introduction/'](https://developer.atlassian.com/cloud/trello/guides/rest-api/api-introduction/) for detailed instructions on how to both create an API_KEY and TOKEN
+
+## TESTING
+Add pytest as a dependency of our project by running poetry add pytest. This should download pytest and also update pyproject.toml for you.
+'''bash
+$ poetry add pytest
+'''
```

Comment:
Worth noting that once one person has run this, the next person shouldn't actually need to. As long as they run the initial `poetry install` command to update their dependencies, this will get pulled in automatically because it's now registered in the `pyproject.toml` file